lesson08/assignment/sp1/src/components/BannerTop.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <header class="header">
+    <div class="header__container">
+      <div class="header__container__logo">
+        <img src="../assets/img/FlagIcon01v1.png" alt="Red Flag Logo">
+      </div>
+      <h1 class="header__container__brandTitle"><div class="firstLetter">S</div>eattle <div class="firstLetter">P</div>laygrounds</h1>
```

Comment:
https://www.w3schools.com/cssref/sel_firstletter.asp This would be a better choice. If you do want to wrap that letter in an element, span would be more appropriate than div.